server/src/operators/search_operator.rs

```diff
@@ -1426,15 +1430,41 @@ pub async fn retrieve_chunks_from_point_ids(
         .map(|point| point.point_id)
         .collect::<Vec<_>>();
 
-    let metadata_chunks =
-        if data.slim_chunks.unwrap_or(false) && data.search_type != SearchMethod::Hybrid {
-            get_slim_chunks_from_point_ids_query(point_ids, pool.clone()).await?
-        } else if data.content_only.unwrap_or(false) {
-            get_content_chunk_from_point_ids_query(point_ids, pool.clone()).await?
-        } else {
-            get_chunk_metadatas_and_collided_chunks_from_point_ids_query(point_ids, pool.clone())
-                .await?
-        };
+    let only_insert_qdrant = std::env::var("ONLY_INSERT_QDRANT")
+        .map(|val| val == "true")
+        .unwrap_or(false);
+
+    let metadata_chunks = if only_insert_qdrant {
+        search_chunk_query_results
+            .search_results
+            .iter()
+            .map(|search_result| {
+                let chunk_html: Option<String> = match search_result.payload.get("content") {
+                    Some(qdrant::Value {
+                        kind: Some(Kind::StringValue(content)),
+                        ..
+                    }) => Some(content.clone()),
+                    _ => None,
+                };
+
+                ChunkMetadataTypes::QdrantChunkMetadata(QdrantChunkMetadata {
+                    qdrant_point_id: search_result.point_id,
+                    chunk_html,
+                    tag_set: None,
+                    images_urls: None,
+                    link: None,
+                    metadata: None,
+                })
+            })
+            .collect()
```

Comment:
Stopping off here, but it should be pretty easy to extract the rest of the payload values needed.